### DISCUSSION TITLE
nr1-github install difficulties

### DISCUSSION THREAD

**Discussion by pbeckhelm** [newrelic/nr1-github#119] [Q&A]
Hi team,

I'm trying to install this via the directions here:  https://onenr.io/0LREV5Lreja

Unfortunately I'm stopped very early in the process and it's not clear what the issue is.  Here's the attempt to clone the repo:

$ nr1 nerdpack:clone -r https://github.com/newrelic/nr1-github.git
 ›   Error: Cannot destructure property 'path' of 'flags' as it is undefined.
 ›   Code: UNKNOWN

Any idea what I can do to understand this error and resolve it?

Best, patrick

**Comment by scrogatl** [comment]
Same issue for me.. both on ubuntu linux
